src/component/index.js

```diff
@@ -4,4 +4,13 @@ import { AddNote } from "./Note/Note";
 import { SavedNote } from "./SavedNote/SavedNote";
 import { TrashNote } from "./TrashNote/TrashNote";
 import { ArchiveNote } from "./ArchiveNote/ArchiveNote";
-export { Navbar, Sidebar, AddNote, SavedNote, TrashNote, ArchiveNote };
+import { EditNote } from "./EditNote/EditNote";
+export {
+  Navbar,
+  Sidebar,
+  AddNote,
+  SavedNote,
+  TrashNote,
+  ArchiveNote,
+  EditNote,
+};
```

Comment:
suggestion: we can export directly like 
' export { ArchiveNote } from "./ArchiveNote/ArchiveNote";  ' instead of importing and then exporting, this wil work for named exports only.

Reply:
Thanks for suggesting this didn't knew about it.will implement it in the follow up PR